test/test_mutable_binheap.jl

```diff
@@ -99,17 +101,20 @@ end
     end
 
     @testset "make mutable binary maxheap" begin
+        vs = [4, 1, 3, 2, 16, 9, 10, 14, 8, 7]
         h = MutableBinaryMaxHeap(vs)
 
         @test length(h) == 10
         @test !isempty(h)
         @test first(h) == 16
         @test isequal(list_values(h), vs)
-        @test isequal(heap_values(h), [16, 14, 10, 8, 7, 3, 9, 1, 4, 2])
+        @test isequal(heap_values(h), [16, 14, 10, 8, 7, 9, 3, 2, 4, 1])
```

Comment:
I think this is just due to the difference in the implementation of `heapify` and the previous method `_heap_bubble_up`. Otherwise, there is no problem. (Let me know if that's not the case)

Reply:
Do these both follow the heap rule about being larger than children?
(I don't know how to read heaps when written in array form TBH)

Reply:
Yes, they both are valid heaps. I also confirmed it using this function [here](https://www.geeksforgeeks.org/how-to-check-if-a-given-array-represents-a-binary-heap/#:~:text=An%20Efficient%20Solution%20is%20to,%2C%20then%20x%20%3E%20z).

Reply:
seems fine then